docs/pbc/all/service-point-management/202311.0/unified-commerce/service-points-feature-overview.md

```diff
@@ -64,6 +65,35 @@ After placing an order, the customer can see the selected service point on the O
 
 ![Storefront order with a service point](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/service-point-management/unified-commerce/service-points-feature-overview.md/storefront-order-service-point.png)
 
+
+## Current constraints
+
+* Currently, only product offers are supported, which means that services can be configured exclusively at the level of a product offer
+* Product Catalog is not supported, meaning it can not be filtered by the service type or service provided in the specific service point
+* Product details page is not supported, meaning it will show all product offers created in the system for the product
```

Comment:
How's this a constraint in the context of service points?

Reply:
We may call it feature gap, basically these are the things that are not yet requested/implemented.